library/styleguideComponents/Examples/index.js

```diff
@@ -0,0 +1,70 @@
+import 'core-js/modules/es.array.map';
+import 'core-js/modules/es.function.name';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Playground from 'rsg-components/Playground';
+import Markdown from 'rsg-components/Markdown';
+import ExamplesRenderer from './ExamplesRenderer';
+
+export default function Examples(_ref1, _ref2) {
+    const { examples, name, exampleMode } = _ref1;
+    const codeRevision = _ref2.codeRevision;
+
+    if (name === 'Customization') {
+        if (examples.length == 1) {
+            return (
+                <ExamplesRenderer name={name}>
+                    <div className="react-rainbow-customization_container">
+                        <Playground
+                            code={examples[0].content}
+                            evalInContext={examples[0].evalInContext}
+                            name={name}
+                            settings={examples[0].settings}
+                            exampleMode={exampleMode}
+                        />
+                    </div>
+                </ExamplesRenderer>
+            );
+        } else {
+            return (
+                <ExamplesRenderer name={name}>
+                    <CustomizationPage {..._ref1} codeRevision={codeRevision} />;
```

Comment:
where is CustomizationPage defined here?
it not imported and not declared in this file